docs/metric-update-procedure/README.md

```diff
@@ -54,23 +88,16 @@ _**Notes**:_
 
 ## 3. Import CSVs to Google Spreadsheets
 
-To update the latest graphs and values for development teams, import each report in the
-`reports_dir` directory to the corresponding Google Sheet. To import a CSV file:
+To import the generated CSVs into the 3 different Google Spreadsheets, execute the following make command:
 
-3.1 Select the corresponding sheet in [Autopush-rs Metrics][1], [FxA Metrics][2] or
-[Merino-py Metrics][3] 
+```shell
+make run_google_sheet_uploader
+```
 
-3.2 Import the CSV 
+This will go through the `reports` folder and takes every CSV file and imports them into the right spreadsheet and tab. This will happen based on the mapping in the `config.ini` file.
 
-- In the top menu bar, select `File > Import`, a file selection dialogue should appear
-- Select the `Upload` tab and click `Browse`, a local file selection dialogue should appear
-- Select the corresponding csv file from the `reports_dir`, an `Import file` dialogue should
-  appear
-- For `Import location`, select `Replace current sheet`, for `Separator type,` select
-  `Detect automatically`, leave the `Convert text to numbers, dates, and formulas` box
-   checked, and click `Import data.` The Google Sheet should now reflect the latest data.
 - If the report being imported is a results or coverage report. Convert the type of the date
-     column to 'plain text' so that the graphs display at an even cadence. 
+     column to 'plain text' so that the graphs display at an even cadence.
```

Comment:
Does this still need to happen? As well as `3.3 Update Trend Table Dates`? If so, we can probably add this to the script as well.

Reply:
I think we can add this once the basic uploading works as expected and is merged into `main`.